packages/components/src/ui/context/use-context-system.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * External dependencies
+ */
+import memoize from 'memize';
+import { kebabCase, uniq } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import warn from '@wordpress/warning';
+
+/**
+ * Internal dependencies
+ */
+import { cx } from '../styles';
```

Comment:
It would be great to expand most of the variable names that are hard to decipher when you don't have the full context:
- `cx`
- `ns`
- `k`
- `nextNs`

Reply:
`cx` matches Emotion's export. It could be expanded to `classnames` but it would conflict with the library by the same name.

`ns` is purely internal but I agree it could be expanded for clarity.

`k` could be expanded to `key` just fine.